bldrs-ai/Share
# Placemark New Features

Refactored and implemented New Placemarks Version

Features:

 - Sprite based placemark system, placemarks viewable at all times in the model if notes are open
 - Add placemark to issue or comment 
 - Persistence in Github
 - Selecting a placemark scrolls notes view to the linked issue or placemark
 
 
 Testing
  - Base URL: [Base URL](https://deploy-preview-1263--bldrs-share-prod.netlify.app/share/v/gh/nickcastel50/test-public/main/index.ifc)
  - Open to Selected Issue Marker: [Open To Selected Issue Marker](https://deploy-preview-1263--bldrs-share-prod.netlify.app/share/v/gh/nickcastel50/test-public/main/index.ifc#m:-7.786,15,-3.683,0,1,0;i:2609600494)
  - Open to Selected Comment Marker With Scroll: [Open To Selected Comment Marker With Scroll](https://deploy-preview-1263--bldrs-share-prod.netlify.app/share/v/gh/nickcastel50/test-public/main/index.ifc#m:-23.726,30,-3.576,0,1,0;i:2609600494;ic:2466408331) 
  
  Select the different markers to see the URL change and current issue / comment note update in the UI. 
  
  To place a placemark, click into an issue thread. If you are logged in, you will see a placemark Icon on the main issue and on the "leave a comment" UI elements. To leave one on an existing issue, go to edit issue and then click the marker icon for the issue. To leave one on a comment, click the marker icon on the new comment UI element. Then, double click anywhere on the model.
